scripts/spatial_to_DistanceMatrix/spatial_to_DistanceMatrix.yml

```diff
@@ -0,0 +1,79 @@
+    type: int
+    example: 3395
+  unit_distance:
+    label: unit for distance measurement
+    description: String value that defines the measurement unit for calculating distances between spatial units (m or km) and estimating their areas (m2 or km2) in the study area.
+    type: options
+    options:
+      - m
+      - km
+    example: "m"
+  spatial_file:
+    label: spatial file
+    description: String path represents the file location of a spatial file containing spatial units, which is used to calculate the distance matrix
+    type: text
+    example: "/scripts/spatial_to_DistanceMatrix/input/Colombian_protected_areas.gpkg"
+  column_group:
+    label: name of grouping attribute   
+    description: String that refers to the name of the column in the spatial file with the levels used to define the spatial units. In cases where there are multiple polygons per level, they will be grouped together, and the attributes of the first listed polygon will be taken into account.
+    type: text[]
+    example: ["id_pa", "name"]
+outputs:
+  spatialunits_studyarea:
+    label: intersect spatial units/study area  
+    description: Represents the map of the analyzed spatial units within the study area.
+    type: application/geo+json
+  spatialunits_studyarea_data:
+    label: spatial units study area data
+    description: Refers to the dataset containing information about the spatial units analyzed within the study area. 
+    type: text/csv
+  mtx_distance_centroid:
+    label: distance centroid matrix
+    description: Distance matrix that represents the distances between the centroids of spatial units within the study area. Each element in the matrix denotes the distance between the centroid of one spatial unit and the centroid of another
+    type: text/csv
+  mtx_distance_nearest:
+    label: distance nearest matrix
+    description: Distance matrix that represents the distances between the closest boundaries of spatial units within the study area. Each element in the matrix denotes the distance between the boundary of one spatial unit and the boundary of another.
+    type: text/csv
+  spatial_unit:
+    label: spatial unit 
+    description: Refers to the column in the spatialunits_studyarea_data that denotes the spatial entity or geographic unit used for analysis. This column contains information about the spatial units that have been grouped based on the levels provided in the column_group input.
+    type: text
+  column_area:
+    label: column area
+    description: Refers to the column in the dataset that contains the estimated area for each of the spatial units under analysis. The estimation is expressed in square units (m2 or km2), as defined by the input unit_distance.
+    type: text
+  area_study_area:
+    label: total study area
+    description: Refers to the total area of the analyzed study area. The estimation of this area is expressed in square units (m2 or km2) as defined by the input unit_distance.
```

Comment:
the input is `distance_unit`, not `unit_distance`

Reply:
done